src/constants.py

```diff
@@ -3,17 +3,17 @@
 
 """File containing constants to be used in the charm."""
 
-MYSQL_ROUTER_APT_PACKAGE = "mysql-router"
-MYSQL_ROUTER_GROUP = "mysql"
-MYSQL_ROUTER_USER = "mysql"
-MYSQL_HOME_DIRECTORY = "/var/lib/mysql"
+CHARMED_MYSQL_SNAP = "charmed-mysql"
+CHARMED_MYSQL_SNAP_REVISION = 48
```

Comment:
TODO: change revision to latest `8.0/edge` revision once [charmed mysql snap PR 21](https://github.com/canonical/charmed-mysql-snap/pull/21) is merged